index.css

```diff
@@ -19,18 +20,18 @@ ul{list-style: none;}
 li{font-size: 1.25em;}
 
 button{
-    font-size: 1.5em;
+    font-size: 20px;
     font-weight: bold;
     padding: 10px 20px;
     border-radius: 10px; /*se usa para redondear los bordes de los botones*/
     border: 2px solid rgb(0,0,0,0.5);
-    color:white;
+    color:rgb(15, 14, 14,0.75);
```

Comment:
recuerda pasar los colores a hexadecimal: https://es.wikipedia.org/wiki/Colores_web

Reply:
The color format has updating in last commit.